### PR TITLE
Fixed objects drifting during Near Manipulation

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Fixed issue with the Object Manipulator causing objects to drift during near manipulationl

## Changes
- Fixes: #10198

### CODE REVIEW

File: Assets/MRTK/SDK/Features/Input/Handlers/Manipulation/ManipulationHandler.cs [hunk 1]
```diff
@@ -684,7 +684,9 @@ private void HandleTwoHandManipulationUpdated()
                 // the initial pointer pose; if far manipulation, a more complex
                 // look-rotation-based pointer pose is used.
                 MixedRealityPose pose = IsNearManipulation() ? new MixedRealityPose(GetPointersCentroid()) : GetAveragePointerPose();
-                targetTransform.Position = moveLogic.Update(pose, targetTransform.Rotation, targetTransform.Scale, true);
+
+                // The manipulation handler is not built to handle near manipulation properly, please use the object manipulator
+                targetTransform.Position = moveLogic.UpdateTransform(pose, targetTransform, true, false);
```
[the remark this answers is not in the document]
Comment (RogPodge) [reply Assets/MRTK/SDK/Features/Input/Handlers/Manipulation/ManipulationHandler.cs:689]: Yep, when you try to use the manipulation handler component, it tries to get you to upgrade to the object manipulator, so it's all deprecated

File: Assets/MRTK/SDK/Features/Input/Handlers/Manipulation/ManipulationHandler.cs [hunk 2]
```diff
@@ -748,13 +750,16 @@ private void HandleOneHandMoveUpdated()
...
+
+            // The manipulation handler is not built to handle near manipulation properly, please use the object manipulator
+            targetTransform.Position = moveLogic.UpdateTransform(pointerPose, targetTransform, rotateInOneHandType != RotateInOneHandType.RotateAboutObjectCenter, false);
             if (constraintOnMovement == MovementConstraintType.FixDistanceFromHead && moveConstraint != null)
             {
                 moveConstraint.ApplyConstraint(ref targetTransform);
             }
 
             float lerpAmount = GetLerpAmount();
+            Debug.Log(lerpAmount);
```
[the remark this answers is not in the document]
Comment (RogPodge) [reply Assets/MRTK/SDK/Features/Input/Handlers/Manipulation/ManipulationHandler.cs:762]: yep

[the remark this answers is not in the document]
Comment (RogPodge) [reply Assets/MRTK/SDK/Features/Input/Handlers/Manipulation/ManipulationHandler.cs:762]: Fixed

**Review by keveleigh — APPROVED** [review]
Seems reasonable to me! I didn't pull this down to test it, but the changes themselves seem good

**Comment by provencher** [comment]
Nice job on the fix!

**Pull request merged**